assets/scss/coder.scss

```
@@ -1,18 +1,18 @@
 @import "css/normalize";
 @import "variables";
 @import "font-awesome/fontawesome";
-@import "font-awesome/regular";
-@import "font-awesome/solid";
-@import "font-awesome/brands";
-@import "font-awesome/v4-shims";
+//@import "font-awesome/regular";
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)